gulpfile.js

```diff
@@ -102,7 +102,8 @@ gulp.task('vendorScript', function() {
       'bower_components/angular-route/angular-route.js',
       'bower_components/jquery/dist/jquery.js',
       'bower_components/ngstorage/ngStorage.js',
-      'bower_components/bootstrap/dist/js/bootstrap.js'
```

Comment:
Direi di sostituire questo array con una variabile in modo da poterlo riusare anche in quello dopo. Anche perche mi pare rischiamo fin da subito di non avere le dipendenze sincronizzate tra development e production

Reply:
Non dovrebbe essere un issue a sé?